cmd/brew-postgresql-upgrade-database.rb

```diff
@@ -49,6 +49,13 @@
   # https://www.postgresql.org/docs/10/static/pgupgrade.html
   ohai "Upgrading #{name} data from #{pg_version_data} to #{pg_version_installed}..."
 
+  # get 'lc_collate' from old DB"
+  unless system "#{old_bin}/pg_ctl", "-w", "-D", datadir, "status", out: File::NULL
+    safe_system "#{old_bin}/pg_ctl", "-w", "-D", datadir, "start", out: File::NULL
+  end
+  lc_collate = `#{old_bin}/psql postgres -qtAc "SELECT setting FROM pg_settings WHERE name LIKE 'lc_collate'";`.strip
+  lc_collate = 'en_US.utf8' if lc_collate.empty?
```

Comment:
```ruby
lc_collate ||= "en_US.utf8"
```
Alternatively, what do you think about just not passing the `lc-collate` flag at all in this case?

Reply:
Sound better. There will less unpredicted side effects. I'll change it so.